.github/workflows/build-from-empty-opam.yaml

```diff
@@ -0,0 +1,37 @@
+name: Build from empty opam switch
+
+# This workflow uses an empty opam switch and installs semgrep-core.
+# It will detect missing opam dependencies in semgrep.opam, as in #7400
+# where the "uri" opam package was not listed in semgrep.opam.
+# The other workflows use a docker container with the opam dependencies
+# already installed (returntocorp/ocaml-layer) for reducing the CI times.
+
+on: [pull_request]
+
+jobs:
+  build:
+    strategy:
+      fail-fast: false
+      matrix:
+        ocaml-version: ["4.14.x"]
```

Comment:
ok, romain changed this 4.14.x to 4.14.0
weird that it was causing the CI job to take forever.